components/threading/global_threads/thread_list/thread_list.tsx

```diff
@@ -146,6 +150,10 @@ const ThreadList = ({
         return {data: true};
     }, [currentTeamId, data, unread, selectedThreadId]);
 
+    const handleCloseMarkAllAsReadModal = useCallback(() => {
+        dispatch(closeModal(ModalIdentifiers.MARK_ALL_THREADS_AS_READ));
+    }, []);
+
```

Comment:
this doesnt have to be inside the useCallback, since this function is not a prop being passed down to a component https://reactjs.org/docs/hooks-reference.html#usecallback

Reply:
Thanks for pointing it out :)
I moved this code into `handleOpenMarkAllAsReadModal`
```ts
const handleOpenMarkAllAsReadModal = useCallback(() => {
    const handleCloseMarkAllAsReadModal = () => {
        dispatch(closeModal(ModalIdentifiers.MARK_ALL_THREADS_AS_READ));
    };

    const handleConfirm = () => {
        handleAllMarkedRead();
        handleCloseMarkAllAsReadModal();
    };
    ...
```